src/libcore/array.rs

```diff
@@ -311,7 +315,9 @@ mod impls_using_const_generics {
 
     #[stable(feature = "try_from", since = "1.34.0")]
     impl<T, const N: usize> TryFrom<&[T]> for [T; N]
-        where T: Copy, [T; N]: LengthAtMost32
+    where
+        T: Copy,
+        [T; N]: LengthAtMost32,
```

Comment:
:nauseated_face:

Reply:
Looks like there's a `where_single_line` option fixing the list-breaking behavior, but it still keeps `where` un-indented, which goes entirely again all sense of style.

Reply:
Style is objective, but since you’ve brought it up: 

1. I don’t find this objectionable at all. I can quickly skim it.
1. Remember that all of the `LengthAtMost32` bounds will disappear someday (ideally). The diff when that happens will be *extremely* easy to review.

Reply:
`Self: LengthAtMost32,`